lib/screens/cart_page.dart

```diff
@@ -655,6 +659,98 @@ class _CartPageState extends State<CartPage> {
     );
   }
 
+  Widget _buildAboutDeliveryMethodInfo() {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(top: BorderSide(color: Color(0xFFA2A2A2))),
+      ),
+      child: Padding(
+        padding: EdgeInsets.symmetric(vertical: 37),
+        child: Column(
+          children: <Widget>[_buildAboutDeliveryMethod()],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildAboutDeliveryMethod() {
+    return Column(
+      children: <Widget>[
+        Container(
+          margin: const EdgeInsets.only(top: 17.0, bottom: 57.0),
+          child: Text(
+            '配送方法について',
+            style: TextStyle(
+              fontSize: 18,
+              color: paletteBlackColor,
+              fontWeight: FontWeight.w600,
+            ),
+          ),
+        ),
+        Container(
+          child: Text(
+              'ゆうパックもしくは宅急便にて配送致します。\n一律500円の送料がかかります。5,000円以上のお買い上げで送料が無料になります。）',
+              textAlign: TextAlign.justify,
+              style: TextStyle(
+                  color: paletteBlackColor,
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w300)),
```

Comment:
Please add a `,` here: 

fontWeight: FontWeight.w300)`,`)

And do auto format https://stackoverflow.com/a/37429293